lib/paths-cache.js

```diff
@@ -387,28 +388,11 @@ export default class PathsCache extends EventEmitter {
     const cmd = "find"
     const args = ["-L", directoryPath + "/", "-type", "f", "-not", "-regex", ignorePattern]
 
-    const stdout = await spawn(cmd, args)
+    const stdout = await exec(cmd, args)
```

Comment:
Why was this not failing? we should probably add a test for `rebuildCache` with `find`

Reply:
I am replacing the whole `find` backend with glob. This will solve many issues, especially on Windows. So, we should wait until I bring that PR.

Reply:
I will work on a PR updating tests to jasmine 3 soon.